mpf/core/utility_functions.py

```diff
@@ -242,7 +242,7 @@ def dict_merge(a, b, combine_lists=True) -> dict:
             elif k in result and isinstance(result[k], dict):
                 result[k] = Util.dict_merge(result[k], v, combine_lists)
             elif k in result and isinstance(result[k], list):
-                if isinstance(v, dict) and v[0] == dict(_overwrite=True):
+                if isinstance(v, list) and v[0] == dict(_overwrite=True):
```

Comment:
I don't remember what situation I was in that kept crashing on this error, but attempting to access `v[0]` when `v` is a dict... seems like just a typo. If I'm overlooking something here, please let me know!